src/server/test/apis/webhook-test.ts

```diff
@@ -0,0 +1,32 @@
+import test from 'ava';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+
+import {handlePullRequestReview} from '../../webhook-events';
+
+const hookJsonDir = path.join(__dirname, '..', 'static', 'webhook-data');
+
+const TEST_SECRETS = {
+  GITHUB_CLIENT_ID: 'ClientID',
+  GITHUB_CLIENT_SECRET: 'ClientSecret',
+  PUBLIC_VAPID_KEY: 'BPtJjYprRvU3TOb0tw3FrVbLww3bp7ssGjX99PFlqIOb3b8uOH4_Q21GYhwsDRwcfToaFVVeOxWOq5XaXD1MGdw',
+  PRIVATE_VAPID_KEY: 'o1P9aXm-QPZezF_8b7aQabivhv3QqaB0yg5zoFs6-qc',
+};
+
+test('Webhook pull_request_review: submitted-state-changes_requested.json', async (t) => {
+  const eventContent = await fs.readJSON(path.join(hookJsonDir, 'pull_request_review', 'submitted-state-changes_requested.json'));
+  await handlePullRequestReview(TEST_SECRETS, eventContent);
+  
+  // For now nothing to test - should just pass
```

Comment:
This should probably just test the boundary of the server. ie. check that the server calls `webpush.sendNotification()` with the correct params.

Reply:
Originally I wanted to just test if our code was called (i.e. the sendNotification() function) and ensure the parameters, but I can't see a way to do it. Typically I'd just use sinon to spy on the function and examine the call count and arguments.

That isn't working in this case, possible because of the typescript build?

Do you have any example of doing this?